seed_stage_based_messaging/utils.py

```diff
@@ -36,7 +44,11 @@ def get_identity_address(identity_uuid, use_communicate_through=False):
         'Authorization': 'Token %s' % settings.IDENTITY_STORE_TOKEN,
         'Content-Type': 'application/json'
     }
-    r = requests.get(url, params=params, headers=headers).json()
+    session = requests.Session()
+    session.mount(settings.IDENTITY_STORE_URL, HTTPAdapter(max_retries=5))
+    result = session.get(url, params=params, headers=headers)
```

Comment:
Should we maybe specify a timeout here? Since we're catching the timeout error in the main task.